tests/suite/scripting/destructuring.typ

```diff
@@ -129,12 +129,12 @@
 
 --- destructuring-let-array-too-few-elements ---
 // Error: 13-14 not enough elements to destructure
-// Hint: 13-14 the provided array has a length of 2
+// Hint: 13-14 the provided array has a length of 2, but the pattern expects 3 elements
 #let (a, b, c) = (1, 2)
 
 --- destructuring-let-array-too-few-elements-with-sink ---
 // Error: 7-10 not enough elements to destructure
-// Hint: 7-10 the provided array has a length of 2
+// Hint: 7-10 the provided array has a length of 2, but the pattern expects 4 elements
```

Comment:
This test shows that the number used is incorrect: `#let (..a, b, c, d) = (1, 2, 3)` works, and `a` is assigned `()` (empty array). Spreads shouldn't count in the expected number of elements.

Reply:
maybe with spreads, we could then even do "expects _at least_ 3 elements"